datasets/beheerkaart/cbs_grid/dataset.json

```diff
@@ -28,93 +28,233 @@
+            "description": "Naam van gemeente die overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "buurtnaam": {
+            "type": "string",
+            "description": "Naam van buurt die overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "buurtcode": {
+            "type": "string",
+            "description": "Code van buurt die overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "dominantbuurt": {
+            "type": "string",
+            "description": "Naam van dominante buurt die overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "dominantbuurtcode": {
+            "type": "string",
+            "description": "Code van dominante buurt die overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "wijknaam": {
+            "type": "string",
+            "description": "Naam van wijk die overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "wijkcode": {
+            "type": "string",
+            "description": "Code van wijk die overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "dominantwijk": {
+            "type": "string",
+            "description": "Naam van dominante wijk die overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "dominantwijkcode": {
+            "type": "string",
+            "description": "Code van dominante wijk die overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "stadsdeelnaam": {
+            "type": "string",
+            "description": "Naam van stadsdeel dat overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "stadsdeelcode": {
+            "type": "string",
+            "description": "Code van stadsdeel dat overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "dominantstadsdeel": {
+            "type": "string",
+            "description": "Naam van dominant stadsdeel dat overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "dominantstadsdeelcode": {
+            "type": "string",
+            "description": "Code van dominant stadsdeel dat overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "ggwGebiedNaam": {
+            "type": "string",
+            "description": "Naam van gebiedsgerichtwerken-gebied dat overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "ggwGebiedCode": {
+            "type": "string",
+            "description": "Code van gebiedsgerichtwerken-gebied dat overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "dominantGgwGebied": {
+            "type": "string",
+            "description": "Naam van dominant gebiedsgerichtwerken-gebied dat overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "dominantGgwGebiedcode": {
+            "type": "string",
+            "description": "Code van dominant gebiedsgerichtwerken-gebied dat overlapt met het BKT-CBS-grid-100-vlak."
+          },
+          "percentageBlauw": {
+            "type": "number",
```

Comment:
Als dit een percentage is, kan je een maximum en minimum waarde 0-100 toevoegen. Dat kunnen we de kolommen aanmaken met het juiste data type en is deze meta-informatie beschikbaar voor afnemers. Zie ook https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#data-types-number.
`    "maximum": 100,
    "minimum": 0
`

Reply:
Ik heb dit voor alle percentagekolommen toegevoegd zoals je voorstelt. Ook heb ik daarvan de descriptions iets aangepast. Daarin zie je andere minimumpercentages: raster_100: [10-100] en raster_10: [1-100]. In de onderliggende gegevens komen ook lagere percentages voor, vanaf 0%, maar in de door ons geselecteerde en uiteindelijk aangeleverde gegevens zoals in de refdb, zie je dat deze laagste percentages er door ons uit zijn weggeselecteerd. Maar in de kolom-metadata lijkt minimum 0% me terecht.